test/e2e/audit.go

```diff
@@ -44,14 +44,26 @@ var _ = describe("Audit", func() {
 	})
 
 	It("Should audit API calls to create, update, patch, delete pods. [Audit] [Zalando]", func() {
+		runAsNonRoot := true
 		pod := &apiv1.Pod{
 			ObjectMeta: metav1.ObjectMeta{
 				Name: "audit-pod",
 			},
 			Spec: apiv1.PodSpec{
 				Containers: []apiv1.Container{{
 					Name:  "pause",
-					Image: "container-registry.zalando.net/teapot/pause:3.4.1-master-18",
+					Image: "container-registry.zalando.net/teapot/pause:3.7-master-21",
+					// Drop all capabilities, run as non-root, and use the default seccomp profile.
+					// This is required as pods created are assumed to be restricted by default since k8s v1.31.
```

Comment:
I don't fully understand. If they are restricted by default, why do we need to add these instructions? Do you have a link to the docs of the change that was made in 1.31.